invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ManageDefaultBrandingAction/ManageDefaultBrandingAction.js

```diff
@@ -0,0 +1,112 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Button, Message, Popup } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { http } from "react-invenio-forms";
+import PropTypes from "prop-types";
+
+export class ManageDefaultBrandingAction extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: undefined,
+    };
+  }
+
+  handleSetBranding = async (value) => {
+    const { recordCommunityEndpoint, updateRecordCallback } = this.props;
+    this.setState({ loading: true });
+    try {
+      const response = await http.put(recordCommunityEndpoint, {
+        default: value,
+      });
+
+      this.setState({ loading: false });
+      updateRecordCallback(response.data);
+    } catch (e) {
+      console.error(e);
+      this.setState({
+        error: e?.response?.data?.message || e?.message,
+        loading: false,
+      });
+    }
+  };
+
+  render() {
+    const { loading, error } = this.state;
+    const { result, isCommunityDefault } = this.props;
+
+    const communityTitle = result.metadata.title;
+    return error ? (
+      <Message
```

Comment:
Would ErrorMessage component from react-invenio-forms work here?

Reply:
Will work here, but it looks more scary IMHO. Take a look (the white box inside is a bug also should be fixed): 
<img width="1040" alt="Screenshot 2023-10-03 at 20 36 24" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/61321254/2cd56c6f-3752-40ed-b8fa-b5ca0654f44f">